auth0/structure_auth0_connection.go

```diff
@@ -485,6 +506,28 @@ func expandConnectionOptionsOIDC(d ResourceData) *management.ConnectionOptionsOI
 	return o
 }
 
+func expandConnectionOptionsSAML(d ResourceData) *management.ConnectionOptionsSAML {
+	o := &management.ConnectionOptionsSAML{
+		SigningCert:     String(d, "signing_cert"),
+		ProtocolBinding: String(d, "protocol_binding"),
+		IdpInitiated: &management.ConnectionOptionsSAMLIdpinitiated{
+			ClientID:             String(d, "client_id"),
+			ClientProtocol:       String(d, "client_protocol"),
+			ClientAuthorizequery: String(d, "client_authorizequery"),
+		},
```

Comment:
I suspect there is an error here. `String(d, "client_id")` should be run inside a `Set(d, "idpinitiated").Elem(...)` block.